Covid19.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import bs4
+from mysql_connector import MYSQL_CONNECTOR
+import sys
+
+
+request = requests.get("https://www.worldometers.info/coronavirus/")
```

Comment:
I think the request process should do a function. For example:

```
def get_request(url):
	request = requests.get(url)
	return request
```

Reply:
Then you can parse the response variable. For example

```
URL = "https://www.worldometers.info/coronavirus/"
response = get_request(URL)

```

Reply:
done in..

https://github.com/mrKYLN/covid19_crawler/pull/1/commits/4fe8dc730d8fbd193e36ae94571b86b01c13071d